geniza/common/management/commands/visual_tests.py

```diff
@@ -20,16 +20,22 @@ def get_browser(self):
         options.add_argument("--headless")
         return webdriver.Chrome(options=options)
 
-    def take_snapshots(self, browser):
+    def take_snapshots(self, browser, dark_mode=False):
```

Comment:
I noticed in the percy docs that [percy script](https://docs.percy.io/docs/percyscript) has been deprecated and we should switch to using the [snapshot command](https://docs.percy.io/docs/cli-snapshot) — we don't need to change it now necessarily, but you are making a lot of changes and I wonder if it would simplify anything here.

I'm ok with your approach, but seeing this makes me think we should switch sooner than later so we don't have to reimplement custom logic twice.

Reply:
Totally agree, I wasn't aware. I'll make a chore for it, since I'd like to get this one out first.

Reply:
That works for me.